panorama_connector.py

```diff
@@ -362,7 +362,7 @@ def _add_commit_status(self, job, action_result):
 
         return phantom.APP_SUCCESS
 
-    def _commit_config(self, action_result, use_partial_commit=False):
```

Comment:
since this param is used in just one location, it would be simpler to grab it inside this function directly from `param`.